app/controllers/api/databases.php

```diff
@@ -2781,3 +2877,114 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
 
         $response->dynamic($usage, Response::MODEL_USAGE_COLLECTION);
     });
+
+App::get('/v1/databases/:databaseId/slow-queries')
+    ->desc('List Slow Queries Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.read')
+    ->label('usage.metric', 'documents.{scope}.requests.read')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'listSlowQueries')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_DOCUMENT_LIST)
+    ->param('databaseId', '', new UID(), 'Database ID.')
+    ->param('queries', [], new ArrayList(new Text(APP_LIMIT_ARRAY_ELEMENT_SIZE), APP_LIMIT_ARRAY_PARAMS_SIZE), 'Array of query strings generated using the Query class provided by the SDK. [Learn more about queries](https://appwrite.io/docs/databases#querying-documents). Maximum of ' . APP_LIMIT_ARRAY_PARAMS_SIZE . ' queries are allowed, each ' . APP_LIMIT_ARRAY_ELEMENT_SIZE . ' characters long.', true)
+    ->inject('response')
+    ->inject('dbForProject')
+    ->action(function ($databaseId, array $queries, Response $response, Database $dbForProject) {
+        $queries = Query::parseQueries($queries);
+        $queries[] = Query::equal('databaseId', [$databaseId]);
+
+        $cursor = Query::getByType($queries, Query::TYPE_CURSORAFTER, Query::TYPE_CURSORBEFORE);
+        $cursor = reset($cursor);
+        if ($cursor) {
+            $documentId = $cursor->getValue();
+            $cursorDocument = $dbForProject->getDocument('slowQueries', $documentId);
+            if ($cursorDocument->isEmpty()) {
+                throw new Exception(Exception::GENERAL_CURSOR_NOT_FOUND, "Slow query '{$documentId}' for the 'cursor' value not found.");
+            }
+            $cursor->setValue($cursorDocument);
+        }
+
+        $response->dynamic(new Document([
+            'documents' => $dbForProject->find('slowQueries', $queries),
+            'total' => $dbForProject->count('slowQueries', Query::groupByType($queries)['filters'], APP_LIMIT_COUNT),
+        ]), Response::MODEL_DOCUMENT_LIST);
+    });
+
+App::get('/v1/databases/slow-queries/:documentId')
```

Comment:
I know we are not using it but lets add the databaseId as a param for consistency

```suggestion
App::get('/v1/databases/:databaseId/slow-queries/:documentId')
```